packages/mana/events_test.go

```diff
@@ -92,8 +92,8 @@ func newRevokeEvent() *RevokedEvent {
 func newUpdateEvent() *UpdatedEvent {
 	return &UpdatedEvent{
 		NodeID:   randomNodeID(),
-		OldMana:  &ConsensusBaseMana{},
-		NewMana:  &ConsensusBaseMana{},
+		OldMana:  &ManaBase{},
+		NewMana:  &ManaBase{},
```

Comment:
This is safer as it already initializes the models.

```suggestion
		OldMana:  NewManaBase(0.0),
		NewMana:  NewManaBase(0.0),
```